devlab/config.ini

```diff
@@ -0,0 +1,55 @@
+# Modify this file with you environment parameters.
+
+public_key_path = .ssh/id_rsa.pub
+# This is relative to user's $HOME, will be replaced with ${HOME}/${cloudferry_path}
+cloudferry_path = Documents
+grizzly_ip = 192.168.1.2
+icehouse_ip = 192.168.1.3
+cloudferry_ip = 192.168.1.4
+
+migrate_key_filename = ~/.ssh/id_rsa
+
+src_auth_url = http://192.168.1.2:35357/v2.0
+src_host = 192.168.1.2
+src_ssh_host = 192.168.1.2
+src_ext_cidr = 192.168.1.0/24
+src_user = admin
+src_password = admin
+src_tenant = admin
+src_temp = /root/temp
+
+src_mysql_user = debian-sys-maint
+src_mysql_password = FkThl0NBhxfPP9VT
+
+src_rabbit_password = guest
+src_rabbit_hosts = 192.168.1.2 
+
+src_compute_host_eph_drv = 192.168.1.2
+
+src_storage_host = 192.168.1.2
+src_storage_protocol_transfer = SSH
+
+dst_auth_url = http://192.168.1.3:35357/v2.0
+dst_host = 192.168.1.3
+dst_ssh_host = 192.168.1.3
+dst_ext_cidr = 192.168.1.0/24
+dst_user = admin
+dst_password = admin
+dst_tenant = admin
+dst_temp = /root/merge
+
+dst_mysql_user = debian-sys-maint
+dst_mysql_password = PLNtql2Ov2CFVtmy
+
+dst_compute_host_eph_drv = 192.168.1.3
+
+dst_rabbit_password = guest
+dst_rabbit_hosts = 192.168.1.3
+
+dst_storage_host = 192.168.1.3
+dst_storage_protocol_transfer = SSH
+
+dst_network_interfaces_for_instance = net04
+
+snapshot_snapshot_path = /root/dump.sql
+snapshot_host = 192.168.1.4
```

Comment:
I think we can heavily reduce the size of config.ini by moving repeated option values, e.g. I see a number of `grizzly_ip` (`192.168.1.2`) repeated, this means we can simply use `<grizzly_ip>` in `config.template` wherever we expect `192.168.1.2`, which means removal of:
- src_auth_url
- src_host
- src_ssh_host
- src_rabbit_hosts
- src_compute_host_eph_drv
- src_storage_host
The same applies for the rest of the options